nginx/generate-nginx-config.sh

```diff
@@ -8,8 +8,12 @@ FABRIC_TEMPLATE_FILE="/etc/nginx/fabric/fabric_nginx-plus.conf.j2"
 if [ "$USE_NGINX_PLUS" = false ];
 then
     FABRIC_TEMPLATE_FILE="/etc/nginx/fabric/fabric_nginx.conf.j2"
+elif [ "$USE_MTLS" = true ];
+then
+    FABRIC_TEMPLATE_FILE="/etc/nginx/fabric/fabric_mtls_nginx-plus.conf.j2"
```

Comment:
We went through an exercise recently where Eduard changed all bash if/else statements to case statements. If I recall correctly the reasoning is that it is more efficient.

In fact, if this is branch cut from master, which is the normal practice, then we are definitely going to have some merge issues.

Reply:
After looking through the rest of the changes, I'm beginning to realize that the branches were cut from release/1.0.1.

That being said, I still prefer to put this feature in the next release and keep 1.0.1 feature frozen.

Reply:
Charles, you are 100% correct, this is from 1.0.1. How would you like to deal with the discrepancies between 1.0.1 and master?

Reply:
Once we merge release/1.0.1 to master, we will create a new release branch from master named release/1.0.2 and this branch should merge without issue.

Reply:
Sounds like a plan